doc/src/site/sphinx/migration_guide.rst

```diff
@@ -24,6 +24,11 @@ From 3.28.1 to 3.30
 - It is now required to explicitly create ``H2OContext`` before you run any of our exposed algorithms. Previously,
   the algorithm would create the H2OContext on demand.
 
+- It is no longer possible to disable web (REST API endpoints) on the client node as we require the Rest API
```

Comment:
This could be mentioned in the section 3.28 -> 3.28.1 and a notion about removal would stay?

Reply:
I think that we are deprecating in 3.28.1 & that option will be completely removed in 3.30

I believe it should be in section 3.28.1 -> 3.30

Reply:
ok, np.